### DISCUSSION TITLE
Is French Extensions planned?

### DISCUSSION THREAD

**Discussion by iPeeZz** [aniyomiorg/aniyomi#274] [General]
Hello, sorry to bother you but I wanted to know if you plan to add French extensions on the app? 
I know some websites that can be added like: 
http://www.mavanimes.co/
http://animeresistance.co/
https://anime-flix.net/
https://voiranime.com/
Thanks for this great app
Have a nice day

**Comment by joelchrono12** [comment]
I think they are planned, but if they haven't added latin spanish dubs yet, I doubt french will come first.

**Comment by mirobane** [comment]
here sommes french website that can be added too 

https://www.ianimes.org/index.php
https://onepiece-streaming.co/
https://www.dragonballsuper-vostfr.com/
https://boruto-vostfr.fr/

have a nice day too

**Comment by Rics-Dev** [comment]
this french extension can be a very good addition too, i watch all my animes in this website:
https://neko-sama.fr/
